Snowdog_Components/components/Molecules/swatch/_swatch.scss

```diff
@@ -10,9 +10,12 @@ $swatch__title-margin                 : 0 0 $spacer !default;
 $swatch__title-font-size              : $font-size-base !default;
```

Comment:
maybe it's good to move these vars into separate file?

Reply:
updated